### PR TITLE
Added tg.ch WMS services

### PR DESCRIPTION
**Pull request by davidoesch**
Added tg.ch WMS services

### CODE REVIEW

**Comment by procrastinatio** [comment]
[Demo link](https://mf-geoadmin3.int.bgdi.ch/ltoed_wms_import_TG/index.html)

**Comment by davidoesch** [comment]
works..thanks
(and cantons love it... we will improve the import)

File: src/js/ImportController.js [hunk 1]
```diff
@@ -139,6 +139,24 @@ goog.require('ga_wmts_service');
...
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/luftbelastung-stickstoff?access_key=YoW2syIQ4xe0ccJA&Service=WMS&Version=1.3.0&Request=GetCapabilities',
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/emissionen-stickoxide?access_key=YoW2syIQ4xe0ccJA&Service=WMS&Version=1.3.0&Request=GetCapabilities',
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/av_wms_mbsf?access_key=YoW2syIQ4xe0ccJA&Service=WMS&Version=1.3.0&Request=GetCapabilities',
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/orthofotos?access_key=YoW2syIQ4xe0ccJA&Service=WMS&Version=1.3.0&Request=GetCapabilities',
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/basisplanf?access_key=YoW2syIQ4xe0ccJA&Service=WMS&Version=1.3.0&Request=GetCapabilities',
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/hintergrundplan?access_key=YoW2syIQ4xe0ccJA&Service=WMS&Version=1.3.0&Request=GetCapabilities',
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/basisplansw?access_key=YoW2syIQ4xe0ccJA&Service=WMS&Version=1.3.0&Request=GetCapabilities',
+        'https://map.geo.tg.ch/proxy/geofy_chsdi3/gewaesserkataster_gewaesser-gewaesserlauf?access_key=YoW2syIQ4xe0ccJAs&Service=WMS&Version=1.3.0&Request=GetCapabilities',
```
Comment (oterral) [inline src/js/ImportController.js]: this one doesn't work

Comment (oterral) [reply src/js/ImportController.js]: @davidoesch can I remove it ?

Comment (davidoesch) [reply src/js/ImportController.js]: remove it

**Pull request merged**